packages/studio-ui/src/store/models/slices/LoadingProgressSlice.ts

```diff
@@ -0,0 +1,4 @@
+export default interface LoadingProgressSlice {
+  loadedStyles: Set<string>;
+  addLoadedStyle: (style: string) => void;
```

Comment:
In the future, we will want to support (at least some) HMR for user styles. As such, I think it could be good to have a dedicated slice for styles specifically. We can add them and remove them as users add `.css` imports (or remove them) to their files. Maybe in the future we have a more complex Loading State and we'd want a Slice like this, but for now I think the Loading Progress can be properly derived without it.

Reply:
Spoke offline, renamed to loadedStyles!